lit.ts

```diff
@@ -119,10 +128,28 @@ function getPkpNftContractAbiPath() {
 	}
 }
 
-async function getPkpHelperContract() {
+async function getPkpHelperV2Contract() {
 	switch (config.network) {
 		case "serrano":
+			return getContract(
+				getPkpHelperV2ContractAbiPath()!,
+				config?.serranoContract?.pkpHelperV2Address as string,
+			);
+		case "cayenne":
+			return getContract(
+				getPkpHelperV2ContractAbiPath()!,
+				config?.cayenneContracts?.pkpHelperV2Address as string,
+			);
+		case "manzano":
+			return getContractFromWorker('manzano', 'PKPHelperV2');
+		case "habanero":
```

Comment:
I think much of the network resolving implementations are already implemented in the PR here: https://github.com/LIT-Protocol/relay-server/pull/37

Reply:
Yeah, that's where the getContractFromWorker function comes from. This implementation is a mirror of the exiting getPKPHelperContract function, which should include the changes in #37 . Am I missing something?